pyanalyze/options.py

```diff
@@ -422,4 +437,16 @@ def _parse_config_section(
                 option_cls = ConfigOption.registry[key]
             except KeyError:
                 raise InvalidConfigOption(f"Invalid configuration option {key!r}")
+            if isinstance(value, bool) and key in all_error_codes and value is True:
+                enabled_error_codes.add(key)
             yield option_cls(option_cls.parse(value, path), module_path)
+
+    if disable_all_default_error_codes:
+        if not enabled_error_codes:
+            warnings.warn(
```

Comment:
This would be the only place we use `warnings.warn` in pyanalyze. I can see a use case: disable all error codes in the whole project, but enable some specific error codes only for some submodules. So I think we should just remove the warning.